src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -3111,7 +3110,7 @@ float Interpolate::InterpolateExecutorBase::coordTransToInput(int outCoord,
         break;
     }
     default: {
-        OPENVINO_THROW("errorPrefix", " does not support specified coordinate transformation mode");
+        OPENVINO_THROW("does not support specified coordinate transformation mode");
```

Comment:
May be at does make sense to enhance the error message at least with the name of the executor class?

Reply:
Done